zerver/webhooks/greenhouse/view.py

```diff
@@ -26,7 +26,7 @@ def dict_list_to_string(some_list: WildValue) -> str:
         elif item_type and item_url:
             internal_template += f"[{item_type}]({item_url}), "
 
-    internal_template = internal_template[:-2]
+    internal_template = internal_template.removesuffix(", ")
```

Comment:
This one probably should be just using `.join(", ")` on an `output_chunks` list. The `internal_template` name is super weird here.